app/controllers/slack_controller.rb

```diff
@@ -0,0 +1,65 @@
+require "uri"
+require "net/http"
+
+class SlackController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  include SlackService
```

Comment:
I'm not a fan really if including classes like this, when all you want is calling methods from the module you're including. This way you can't tell from the code which methods are coming from the included module.

As an alternative you can just use `SlackService.create_post`. Much clearer imo.

To be able to do this you do have to change the service definition to either:

```rb
class SlackService
  def self.create_post(...)
  end
end
```

or

```rb
class SlackService
  class << self
    def create_post(...)
    end
  end
end
```

Reply:
I already found it annoying while writing it :), fixed.